transformer_engine/pytorch/module/grouped_linear.py

```diff
@@ -74,12 +62,9 @@ def forward(
         fp8_meta: Dict[str, Any],
         fuse_wgrad_accumulation: bool,
         cpu_offloading: bool,
-        tp_group: Union[dist_group_type, None],
-        tp_size: int,
         sequence_parallel: bool,
-        tensor_parallel: bool,
         activation_dtype: torch.dtype,
-        parallel_mode: Union[str, None],
+        offsets: Dict[str, int],
```

Comment:
Something like `fp8_meta_offsets` may be more readable:
```suggestion
        fp8_meta_offsets: Dict[str, int],
```